### ISSUE TITLE
Possible documentation issue regarding NODE_ENV

### ISSUE THREAD

**Issue by brentpayton** [ericf/express-handlebars#202]
Should the line

process.env.NODE_ENV === "dev";

be 

process.env.NODE_ENV = "dev";

?

**Comment by pietrofxq** [comment]
No. You should not assign NODE_ENV in your code, this value comes from the system path variables. The documention it's explaining something that happens only on dev mode, OR: process.env.NODE_ENV === "development"
